frontend/src/modules/schemas/editor/stores/editorStore.ts

```
@@ -1,15 +1,17 @@
 import { type HttpBody, useHttpClient } from '@/composables';
-import { useEditorSaveDispatcher } from '../composables';
-import { setObjectParent } from '../models';
+import { useEditorHistory, useEditorSaveDispatcher } from '../composables';
+import { setSchemaRelations } from '../models';
 
 type UpdateSchemaRequest = Partial<Omit<ISchema, 'id'>>;
 
 export const useEditorStore = defineStore('schemas/editor', () => {
   const httpClient = useHttpClient();
-  const schema = ref<ISchema>(null!);
+  const schema: Ref<ISchema> = ref(null!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)